### DISCUSSION TITLE
Neutral color for colored components

### DISCUSSION THREAD

**Discussion by notmedia** [heroui-inc/heroui#671] [Feature requests]
I think that NextUI need to have neutral color option in all colored components

For example:

<img width="553" alt="image" src="https://user-images.githubusercontent.com/12993018/185228063-94bdad71-2c00-44b0-a7c0-9424d1a5ff5f.png">

Here I'm using gradient color because with this config I can get less colored component.
I want to have white text, gray border, gray highlight in dropdown menu, and gray animation on click.

Gray is only for example here, I mean it need to be something neutral colored.

In other words there are no options without customisation make less colored interfaces.

**Comment by jrgarciadev** [comment]
Hey @notmedia actually the `Dropdown` component supports a `neutral` color, here is the doc, `color="default"` https://nextui.org/docs/components/dropdown#variants

Here's the neutral color reference:
https://nextui.org/docs/theme/default-theme#brand

Here's the example screenshot: 

![Screen Shot 2022-09-13 at 22 52 32](https://user-images.githubusercontent.com/30373425/190040816-cf30be61-6d81-4c37-8399-4c33f4b51eac.png)

**Reply by notmedia** [reply]
Hi! Cool, but with this setup on button click it's highlighted with primary color

**Reply by notmedia** [reply]
And also how I could setup this animation color if the color is custom?

**Reply by notmedia** [reply]
And another thing - if I set gradient color then combobox items highlited with primary color

<img width="546" alt="image" src="https://user-images.githubusercontent.com/12993018/190898612-543f065f-77b3-482f-8389-db5450a9b31e.png">

**Reply by jrgarciadev** [reply]
Hey @notmedia :

1. The button is `primary` because the `neutral` colour on `buttons` is not available by default at the moment
2. Do you mean the `drip` animation?, here's an example:
    ```js
    <Button
      css={{
        ".nextui-drip .nextui-drip-filler": {
           fill: "PUT_YOUR_COLOR_HERE",
        }
      }}
    >My Button</Button>
    ```
3. Dropdown doesn't have a `gradient` colour by default, here's the available colours https://nextui.org/docs/components/dropdown#variants

**Reply by notmedia** [reply]
Hi @jrgarciadev!

Thank you, drip animation setup exactly what I want :)
Also would be great if buttons will have neutral color as a feature request)
